src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/WindowsServiceLifetime.cs

```diff
@@ -84,15 +102,20 @@ public Task StopAsync(CancellationToken cancellationToken)
             return Task.CompletedTask;
         }
 
-        // Called by base.Run when the service is ready to start.
+        /// <summary>
+        /// Called by base.Run when the service is ready to start.
```

Comment:
This should be something like "Raises the Start event." (or whatever event it raises) - see https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Event-related. Or maybe these don't raise events?

Reply:
CC @tarekgh @maryamariyan. These are raising events, right? The WindowsServiceLifetime summaries were ones I pulled from existing comments

Reply:
The comment on the code already telling about that:

https://github.com/dotnet/runtime/blob/415a41770bdf8efd4c3217e2c281233ee5cd03ea/src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/WindowsServiceLifetime.cs#L87